webapp/graphql/Search.js

```diff
@@ -0,0 +1,24 @@
+import gql from 'graphql-tag'
+import { userFragment, postFragment } from './Fragments'
+
+export const findResourcesQuery = gql`
+  ${userFragment}
+  ${postFragment}
+
+  query($query: String!) {
+    findResources(query: $query, limit: 5) {
+      __typename
+      ... on Post {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T15:58:49Z" title="Friday, December 20th 2019, 4:58:49 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,27 @@
+import gql from 'graphql-tag'
+
+export const findResourcesQuery = gql`
+  query($query: String!) {
+    findResources(query: $query, limit: 5) {
+      __typename
+      ... on Post {
```


We could use fragments here

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T19:20:50Z" title="Friday, December 20th 2019, 8:20:50 pm +01:00">Dec 20, 2019</time>_

---


Done